graspologic/match/qap.py

```diff
@@ -416,9 +416,8 @@ def _quadratic_assignment_faq(
     # check outlier cases
     if n == 0 or partial_match_value.shape[0] == n:
         # Cannot assume partial_match is sorted.
-        partial_match_value = np.row_stack(
-            sorted(partial_match_value, key=lambda x: x[0])
-        )
+        sort_inds = np.argsort(partial_match_value[:, 0])
```

Comment:
@asaadeldin11 if you have the time (if not no worries): 1. does this fix seem equivalent to you and 2. was there a test for this edge case anywhere? I remember this coming up but dont remember if it had a test associated